Sources/YCoreUI/Extensions/Foundation/String+Localizable.swift

```diff
@@ -9,10 +9,11 @@
 import Foundation
 
 public extension String {
-    /// Gets a localized string resource using the string's current value as key
-    /// - Parameter bundle: the bundle containing the localized string resource to use. Default = main app bundle.
+    /// Gets a localized string resource using the string's current value as key    /// - Parameters:
+    ///   - bundle: the bundle containing the localized string resource to use. Default = main app bundle.
+    ///   - tableName:  the name of the `.strings` file containing the localized strings for this enum.
```

Comment:
```suggestion
    ///   - tableName: the name of the `.strings` file containing the localized strings for this enum.
```
delete extra whitespace